es-ds-docs/nuxt.config.ts

```diff
@@ -7,6 +7,19 @@ export default defineNuxtConfig({
         '@energysage/es-bs-base/scss/bootstrap.scss',
     ],
 
+    // The following will still not work until https://github.com/alibaba/formily/pull/4195
+    vite: {
+        css: {
+            preprocessorOptions: {
+                scss: {
+                    silenceDeprecations: [
+                        'mixed-decls', // https://sass-lang.com/documentation/breaking-changes/mixed-decls/
```

Comment:
This passes the same argument to vite that was passed in `es-bs-base` above to silence in development, however there's a pending PR upstream that must be merged before this argument will work